concatenator_node:
  ros__parameters:
    simulated=False
